docs/faq.md

```diff
@@ -66,13 +66,13 @@ out of the box. Since Neutrino uses webpack-chain and enforces naming of plugins
 object that corresponds with this "schema". Typically this would involve transforming your webpack configuration to nest
 entities requiring a name into an object which maps the name to the entity.
 
-In short, you must transform your webpack configuration in order to merge it into the Neutrino configuration cleanly.
+In short, you must transform your webpack configuration to facilitate its merging into the Neutrino configuration cleanly.
```

Comment:
I think the existing wording was more accurate here. I mentioned this in the previous review comment, which was marked as resolve with no discussion but the change not made?
https://github.com/neutrinojs/neutrino/pull/1592#discussion_r456029776

Reply:
Marking review as resolved was quite a mistake, I should have kept open for discussion. I would commit the changes, as now I think prior was better.